windows-driver-docs-pr/install/devpkey-device-basecontainerid.md

```diff
@@ -89,22 +89,8 @@ For more information about container IDs, see [Container IDs](./container-ids.md
 
 ## Requirements
 
-<table>
-<colgroup>
-<col width="50%" />
-<col width="50%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td align="left"><p>Version</p></td>
-<td align="left"><p>Available in Windows 7 and later versions of Windows.</p></td>
-</tr>
-<tr class="even">
-<td align="left"><p>Header</p></td>
-<td align="left">Devpkey.h (include Devpkey.h)</td>
-</tr>
-</tbody>
-</table>
+**Version**: Windows 7 and later versions of Windows
+**Header**: Devpkey.h (include Devpkey.h)

```

Comment:
Doesn't there need to be an empty line or something between these so they appear on separate lines?  The preview right now show them appearing on the same line (same applies to the other pages):
![image](https://user-images.githubusercontent.com/21045076/174916199-a024de26-5e21-453a-9578-e13145dc7653.png)